examples/all-clusters-app/linux/ValveControlDelegate.cpp

```diff
@@ -55,19 +58,19 @@ void ValveControlDelegate::HandleRemainingDurationTick(uint32_t duration)
 {
     ChipLogProgress(NotSpecified, "Valve remaining duration ticking: %dsec level: %d duration %d", duration, sLevel,
                     sLastOpenDuration);
+    auto valveCluster = ValveConfigurationAndControl::FindClusterOnEndpoint(kValveEndpoint);
```

Comment:
While this is correct, lets try to keep old API functional for now.

Reply:
Done.